resources/assets/js/components/PackageDetailFrame.vue

```diff
@@ -666,11 +663,50 @@ export default {
                     alert('Error: ' + response.message);
                 }
             );
-        }
-    },
+        },
+
+        fallbackCopyTextToClipboard() {
+            /* Get the text field */
+            var textArea = document.createElement("textarea");
+            textArea.value = this.composerString;
+
+            // Avoid scrolling to bottom
+            textArea.style.top = "0";
+            textArea.style.left = "0";
+            textArea.style.position = "fixed";
+
+            document.body.appendChild(textArea);
+            textArea.focus();
+            textArea.select();
+
+            try {
+                var successful = document.execCommand('copy');
```

Comment:
It's deprecated but it's because it's the backup copy, if a user is on an older browser then it will use backup if they don't support the modern method.

We probably could just remove the fallback entirely as 99% of deva will be running the latest version of their browser.